zarr/tests/test_storage.py

```diff
@@ -1802,12 +1802,24 @@ def test_context_manager(self):
             store[self.root + 'baz'] = b'qux'
             assert 2 == len(store)
 
+    def test_delitem(self):
+        store = self.create_store()
+        store[self.root + 'foo'] = b'bar'
+
+        del store[self.root + 'foo']
+
+        assert store[self.root + 'foo'] == b''
+
```

Comment:
This is great. Would suggest moving this method to [the `StoreTests` class above]( https://github.com/I-am-Emmanuel/zarr-python/blob/0144f953e99ec73f4d03a7e6825ea48578e9e600/zarr/tests/test_storage.py#L110 )

Reply:
Alright. I will work on it.

Reply:
This is done